### PR TITLE
Add Akita icons, expand README and include install steps

### LINKED ISSUE #22: Document how to install Akita on chrome and firefox
Since we likely won't have time to deploy to the extension/add-on stores, we should give clear instructions on how to manually install Akita.

### PR DESCRIPTION
**Pull request by sharon-wang**
Fixes: https://github.com/dog-s/akita/issues/22

- Add Akita icons
- Add Akita backstory in README
- Add instructions on how to install Akita

Please preview at: https://github.com/sharon-wang/akita/tree/addDocs and https://github.com/sharon-wang/akita/blob/addDocs/docs/InstallAkita.md.

### CODE REVIEW

File: docs/InstallAkita.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Installing Akita
+
+Akita has been tested in Mozilla Firefox and Google Chrome. Instructions to install on these browsers is provided below. We hope to release Akita to browser extension stores in the near future.
+
+## Step 1: Download Akita Source
+From your preferred directory, run the command:
+
+`git clone https://github.com/dog-s/akita.git`
+
+Note the path to the `akita` directory.
```
Comment (vezwork) [inline docs/InstallAkita.md]: suggestion: "...to the newly created `akita` directory."

File: docs/InstallAkita.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Step 2: Install Akita in Your Browser
+
+### Installing Akita in Google Chrome
+1. In your Chrome address bar, type in `chrome://extensions/`. Press enter.
```
Comment (vezwork) [inline docs/InstallAkita.md:15]: Before this step, the user must make sure the "Developer mode" toggle in the top-right of `chrome://extensions/` is enabled.

File: docs/InstallAkita.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+2. Click on "Load unpacked" and select the `akita` directory.
```
Comment (vezwork) [inline docs/InstallAkita.md]: suggestion: "... and a system file selection window should open. Navigate to the `akita` directory and select it."

File: docs/InstallAkita.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+3. Akita is now installed!
```
Comment (vezwork) [inline docs/InstallAkita.md]: suggestion: "... You should see the Akita extension icon in your browser bar!"

File: docs/InstallAkita.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+### Installing Akita in Mozilla Firefox
+1. In your Firefox address bar, type in `about:debugging#/runtime/this-firefox`. Press enter.
+2. Click on "Load Temporary Add-on..." and select the `manifest.json` file in the `akita` directory.
+3. Akita is now installed!
```
Comment (vezwork) [inline docs/InstallAkita.md]: suggestion: "... You should see the Akita extension icon in your browser bar!"

**Review by vezwork — CHANGES_REQUESTED** [review]

**Comment by sharon-wang** [comment]
@vezwork Updated instructions! Please review :)

**Review by vezwork — APPROVED** [review]
Looks great!!

**Pull request merged**